specs/windows/shimcache.table

```diff
@@ -0,0 +1,12 @@
+table_name("shimcache")
+description("Application Compatibility Cache, contains artifacts of execution.")
+schema([
+    Column("entry", INTEGER, "Execution order"),
```

Comment:
nitpick, be consistent with punctuation for your column descriptions, `entry` has no period, the other columns do.

Reply:
fixed